implementations/rust/ockam/ockam_command/src/enroll.rs

```diff
@@ -357,3 +360,27 @@ impl Auth0Service {
         Ok(())
     }
 }
+
+async fn update_enrolled_identity(
+    ctx: &Context,
+    opts: &CommandGlobalOpts,
+    node_name: &str,
+) -> Result<()> {
+    let mut identities = opts.state.identities.list()?;
+    identities.push(opts.state.identities.default()?);
+
+    let node_state = opts.state.nodes.get(node_name)?;
+    let node_identity = node_state.config.identity(ctx).await?;
+
+    for mut identity in identities {
+        if node_identity.identifier() == &identity.config.identifier {
+            identity.config.enrollment_status = Some(EnrollmentStatus {
+                is_enrolled: true,
+                created_at: SystemTime::now(),
+            });
```

Comment:
Since the `EnrollmentStatus` variables are constants, you could add a constructor so you can write this line like `identity.config.enrollment_status = Some(EnrollmentStatus::enrolled());`.